app/views/docs/functions-deploy.phtml

```diff
@@ -0,0 +1,216 @@
+    <li class="margin-bottom-small">Navigate to the Appwrite Console's <b>Functions</b> page.</li>
+    <li class="margin-bottom-small">Navigate to the <b>Domains</b> tab.</li>
+    <li class="margin-bottom-small">In the table, you'll find a link formatted similar to <code>https://64d4d22db370ae41a32e.appwrite.global</code>. This is your preview.</li>
+</ol>
+
+<p>
+    You can also add a custom domain, which allows you to build custom REST APIs using nothing but Appwrite Functions.
+    To do this, you need to first buy and register a domain.
+    After obtaining a domain, follow these steps to add the domain to Appwrite.
+</p>
+
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li class="margin-bottom-small">Navigate to the Appwrite Console's <b>Functions</b> page.</li>
+    <li class="margin-bottom-small">Navigate to the <b>Domains</b> tab.</li>
+    <li class="margin-bottom-small">Click on <b>Create domain</b>.</li>
+    <li class="margin-bottom-small">Input your domain in the <b>Domain</b> input field and click <b>Next</b>.</li>
+    <li class="margin-bottom-small">Copy the <b>CNAME</b> record provided to you, and add it to your domain registrar.</li>
+    <li class="margin-bottom-small">Click <b>Go to console</b> and wait for the domain name to be verified and certificate to generate.</li>
+</ol>
+
+<p>
+    DNS records can take up to 48 hours to propagate after they're added. 
+    Please retry verification over the next 48 hours. 
+    If the domain verification still fails and you have confirmed DNS records are added correctly, please contact support.
+</p>
+
+<h2><a href="#debugging" id="debugging">Debugging Build</a></h2>
+<p>After deploying a function, you can find the status of the deployment and build logs in the Appwrite Console.</p>
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>In Appwrite Console, navigate to <b>Functions</b>.</li>
+    <li>Click to open a function you wish to inspect.</li>
+    <li>Under the <b>Deployments</b> tab, you'll find the status of the current active deployment and previous inactive deployments.</li>
+    <li>You can access build logs for the active deployment by clicking the <b>Build logs</b> button. You can click on an inactive function's three dots button to find their build logs.</li>
+</ol>
+
+<h2><a href="#redeploy" id="redeploy">Redeploy Builds</a></h2>
+<p>
+    After updating the configuration of your Appwrite Function, you need to redeploy your function for the changes to take effect.
+    You can also redeploy builds to retry failed builds.
+</p>
+<ol class="margin-top margin-bottom-large text-size-normal">
+    <li>In Appwrite Console, navigate to <b>Functions</b>.</li>
+    <li>Click to open a function you wish to inspect.</li>
+    <li>Under the <b>Deployments</b> tab, you'll find the status of the current active deployment.</li>
+    <li>You can redeploy by clicking the <b>Redeploy</b> button.</li>
```

Comment:
Lets mention that you can redeploy failed execution by clicking 3 dots and finding same Redeploy button